app/uk/gov/hmrc/channelpreferences/config/AppConfig.scala

```diff
@@ -27,4 +27,5 @@ class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig)
 
   val auditingEnabled: Boolean = config.get[Boolean]("auditing.enabled")
   val graphiteHost: String = config.get[String]("microservice.metrics.graphite.host")
+  val cdsBaseUrl: String = config.get[String]("microservice.metrics.graphite.host")
```

Comment:
cds has nothing to do with graphite.
it should be microservice.services.customs-data-store
You should use baseUrl method from service-config to get url not config.get
[https://github.com/hmrc/bootstrap-play/blob/7bae9ce33c27fbbf2677d2e39312698008b6885b/bootstrap-common-play-26/src/main/scala/uk/gov/hmrc/play/bootstrap/config/ServicesConfig.scala#L38](baseUrl)

Reply:
I have removed this line it is not used. I am using the baseURL method.